plugins/modules/zos_lineinfile.py

```diff
@@ -37,7 +37,7 @@
         PS (sequential data set), member of a PDS or PDSE, PDS, PDSE.
       - The USS file must be an absolute pathname.
       - Generation data set (GDS) relative name of generation already
-        created. C(e.g. SOME.CREATION(-1).)
+        created. C(e.g. SOME.CREATION(-1\).)
```

Comment:
This is a raw string , is it necessary to escape the closing paren:
```
(-1\).)
```
Could this be the source of some of the hanging closing parens we saw during doc generation.

Reply:
fixed